app/views/active_storage/blobs/_blob.html.erb

```diff
@@ -1,6 +1,6 @@
 <figure class="attachment attachment--<%= blob.representable? ? "preview" : "file" %> attachment--<%= blob.filename.extension %>">
   <% if blob.representable? %>
-    <%= image_tag Rails.application.routes.url_helpers.rails_representation_url(blob.representation(resize_to_limit: local_assigns[:in_gallery] ? [ 800, 600 ] : [ 1024, 768 ]).processed, host: ENV['APP_HOST']) %>
+    <%= image_tag Rails.application.routes.url_helpers.rails_representation_url(blob.representation(resize_to_limit: local_assigns[:in_gallery] ? [ 800, 600 ] : [ 1024, 768 ]), host: ENV['APP_HOST']) %>
```

Comment:
@donrestarone 
The `processed` method creates and uploads the variant of attached image to the service before returning its url which is suggested not to be inline in a template. Removing `processed` will delay the processing (uploading to service).
Thus we can remove this.

reference: https://edgeapi.rubyonrails.org/classes/ActiveStorage/Variant.html
![image](https://user-images.githubusercontent.com/25191509/164952532-c5fea957-2eab-477a-b144-d2c64b2839b5.png)

Reply:
@alis-khadka sounds good, thanks for the quick fix.